dashboards-observability/public/components/synthetics/home_view.tsx

```diff
@@ -12,60 +14,206 @@ import {
   // below states used for date/time picker
   const [recentlyUsedRanges, setRecentlyUsedRanges] = useState<DurationRange[]>([]);
-  const [start, setStart] = useState<ShortDate>('now-4h');
+  const [start, setStart] = useState<ShortDate>('now-2d');
   const [end, setEnd] = useState<ShortDate>('now');
   // stores time value added onto queries
-  const [timeVal, setTimeVal] = useState(" | where startTime > \'" + (moment(convertDateTime(start)).add(8, 'hours')).format(PPL_DATE_FORMAT) 
-                                        + "\' and startTime < \'" + (moment(convertDateTime(end)).add(8, 'hours')).format(PPL_DATE_FORMAT)  + "\'");
+
+  // TODO: make sure both flyouts cannot be visible at the same time
+  const showHelpFlyout = () => {
+    setHelpIsFlyoutVisible(true);
+  };
+
+  const closeHelpFlyout = () => {
+    setHelpIsFlyoutVisible(false);
+  };
+
+  const showTestDetailsFlyout = (item: Record<string, any>) => {
+    setTestDetailsObj(item);
+    setTestDetailsFlyoutVisible(true);
+  };
+
+  const closeTestDetailsFlyout = () => {
+    setTestDetailsObj({});
+    setTestDetailsFlyoutVisible(false);
+  };
+
+  let helpFlyout;
+  if (isHelpFlyoutVisible) {
+    helpFlyout = <PPLReferenceFlyout module="synthetics" closeFlyout={closeHelpFlyout} />;
+  }
+
+  let testDetailsFlyout;
+  if (testDetailsFlyoutVisible) {
+    testDetailsFlyout = (
+      <TestDetailsFlyout testSuiteObject={testDetailsObj} closeFlyout={closeTestDetailsFlyout} />
+    );
+  }
+
+  /*
+   * Settings for the up-time table of all test suites
+   */
+
+  // columns for the History table
+  const columns = [
+    {
+      field: TEST_SUITE_NAME_FIELD,
+      name: 'Test Suite',
+      render: (testSuite: string) => (
+        <EuiLink href={`#${testSuite}`} target="_blank">
+          {testSuite}
+        </EuiLink>
+      ),
+      mobileOptions: {
+        show: false,
+      },
+      sortable: true,
+    },
+    {
+      field: STATUS_FIELD,
+      name: 'Status',
+      sortable: true,
+    },
+    {
+      field: URL_FIELD,
+      name: 'URL',
+      render: (url: string) => (
+        <EuiLink href={`${url}`} target="_blank">
+          {url}
+        </EuiLink>
+      ),
+      mobileOptions: {
+        show: false,
+      },
+      sortable: true,
+    },
+    {
+      field: STATUS_CODE_FIELD,
+      name: 'Status Code',
+      sortable: true,
+    },
+    {
+      field: DOWNLOAD_TIME_FIELD,
+      name: 'Latency/Download time',
+      sortable: true,
+    },
+    {
+      field: START_TIME_FIELD,
+      name: 'Latest Time Polled',
+      type: 'date',
+      render: (date: Date) => {
+        const datet: Date = new Date(date);
+        // needed to convert date into local time, as the index stores times in UTC and this page
+        // incorrectly reads it as local time (while being UTC) so the time will be off by the timezoneoffset
+        const utcdate: Date = new Date(datet.getTime() - datet.getTimezoneOffset() * 60 * 1000);
```

Comment:
i think moment.js has something to convert timezone, rather than calculating it here. Also this is still UTC, do we want to display in user's local time on UI?

Reply:
Will make this consistent with other plugin implementations